xml/System/Index.xml

```diff
@@ -109,9 +132,9 @@
         <Parameter Name="other" Type="System.Index" />
       </Parameters>
       <Docs>
-        <param name="other">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="other">An Index object to compare with this Index object.</param>
+        <summary>Indicates whether the current Index object is equal to another Index object.</summary>
+        <returns><see langword="true" /> if the current Index object is equal to the passed Index object, <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the current Index object is equal to <paramref name="other" />; <see langword="false" /> otherwise.</returns>
```